tests/GlobTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace WP_Rocket\Tests;
+
+trait GlobTrait {
+
+	public function deleteDomainCallback( $root ) {
+		$root = rtrim( $root, '*' );
+		$this->delete_files( $root );
+	}
+
+	/**
+	 * Recursively deletes all the files in the given virtual directory.
+	 *
+	 * @param string $dir Virtual directory absolute path.
+	 */
+	protected function deleteFiles( $dir ) {
+		foreach ( $this->scandir( $dir ) as $item ) {
+			if ( $this->filesystem->is_dir( $item ) ) {
+				$this->delete_files( $item );
```

Comment:
And here too?
```suggestion
				$this->deleteFiles( $item );
```

Reply:
Good catch @Screenfeed. Committed your suggestion https://github.com/wp-media/wp-rocket/pull/2470/commits/c5c8f18c1da98d08b1617ce23032dbaa2ffe87ec